simpleSequential/test_andFunction

```diff
@@ -0,0 +1,12 @@
+import unittest
+from andFunction import andFunction
+
+class TestAndFunction(unittest.TestCase):
```

Comment:
Do you think 2 tests are enough? How about 4 to cover all cases?

Reply:
I think 2 is good in this problem.